code/modules/urist/modules/autolathe_datums.dm

```diff
@@ -10,3 +10,82 @@
 	path = /obj/item/weapon/scissors
 	category = "General"
 
+/obj/item/weapon/clipboard/matter = list("wood" = 30)
+
+/datum/autolathe/recipe/clipboard
+	name = "clipboard"
+	path = /obj/item/weapon/clipboard
+	category = "General"
+
+/datum/autolathe/recipe/rollingpin
+	name = "rolling pin"
+	path = /obj/item/weapon/material/kitchen/rollingpin
+	category = "General"
+
+/obj/item/weapon/pickaxe/hand/matter = list(DEFAULT_WALL_MATERIAL = 1250, "wood" = 500)
+
+/datum/autolathe/recipe/handpickaxe
+	name = "hand pickaxe"
+	path = /obj/item/weapon/pickaxe/hand
+	category = "Tools"
+
+/datum/autolathe/recipe/pickaxe
+	name = "pickaxe"
+	path = /obj/item/weapon/pickaxe/old
+	category = "Tools"
+
+/datum/autolathe/recipe/saw
+	name = "hand saw"
+	path = /obj/item/weapon/carpentry/saw
+	category = "Tools"
+
+/datum/autolathe/recipe/axe
+	name = "wood axe"
+	path = /obj/item/weapon/carpentry/axe
+	category = "Tools"
+
+/datum/autolathe/recipe/woodminihoe
+	name = "wood minihoe"
+	path = /obj/item/weapon/material/minihoe/wood
+	category = "Tools"
+
+/datum/autolathe/recipe/woodhatchet
+	name = "wood handled hatchet"
+	path = /obj/item/weapon/material/hatchet/wood
+	category = "Tools"
+
+/datum/autolathe/recipe/woodcup
+	name = "wooden cup"
+	path = /obj/item/weapon/reagent_containers/food/drinks/woodcup
+	category = "General"
+
+/datum/autolathe/recipe/wooddice
+	name = "wooden d6"
+	path = /obj/item/weapon/dice/wood
+	category = "General"
+
+/datum/autolathe/recipe/woodmop
+	name = "wooden mop"
+	path = /obj/item/weapon/mop/wood
+	category = "Tools"
+
+/datum/autolathe/recipe/woodspade
+	name = "wood handled spade"
+	path = /obj/item/weapon/shovel/spade/wood
+	category = "Tools"
+
+/datum/autolathe/recipe/shovel
+	name = "shovel"
+	path = /obj/item/weapon/shovel
+	category = "Tools"
+
+/datum/autolathe/recipe/cane
```

Comment:
Good idea! I couldn't think of anything to add the hacked lathe.

On 2016-03-28, at 5:47, scrdest <notifications@github.com> wrote:

> In code/modules/urist/modules/autolathe_datums.dm:
> 
> > +/datum/autolathe/recipe/woodmop
> > +	name = "wooden mop"
> > +	path = /obj/item/weapon/mop/wood
> > +	category = "Tools"
> > +
> > +/datum/autolathe/recipe/woodspade
> > +	name = "wood handled spade"
> > +	path = /obj/item/weapon/shovel/spade/wood
> > +	category = "Tools"
> > +
> > +/datum/autolathe/recipe/shovel
> > +	name = "shovel"
> > +	path = /obj/item/weapon/shovel
> > +	category = "Tools"
> > +
> > +/datum/autolathe/recipe/cane
> How about adding a canesword as a hacked recipe? You can already get superior weapons if you've got access to it, anyway.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>